### ISSUE TITLE
Check form- "Select functions" is stuck at the bottom

### ISSUE THREAD

**Issue by nairabhishek73** [DalgoT4D/webapp#689]
<img width="1470" alt="Screenshot 2024-03-25 at 8 42 13 AM" src="https://github.com/DalgoT4D/webapp/assets/119285990/43f5a76b-43ea-454f-8550-c90b785f29f5">

**Comment by fatchat** [comment]
we should remove obvious instructions which say "select something from the list in front of you" or "click one of these buttons" or "fill out this form" - users know what forms on websites are

**Comment by nairabhishek73** [comment]
Agreed. You can go ahead and take this text out overall if simpler.
